tests/disaster-recovery/regional-dr/test_failover_and_relocate.py

```diff
@@ -201,6 +244,46 @@ def test_failover_and_relocate(
         logger.info(f"Waiting for {wait_time} minutes to run IOs")
         sleep(wait_time * 60)
 
+        # Get lastGroupSyncTime after failover.
+        # The parameter lastGroupSyncTime may not be present for some time after failover.
+        for drpc_data in TimeoutSampler(300, 5, drpc_obj.get):
+            post_failover_last_group_sync_time = drpc_data.get("status").get(
+                "lastGroupSyncTime"
+            )
+            if post_failover_last_group_sync_time:
+                logger.info("After failover - Obtained lastGroupSyncTime.")
+                # Adding an additional check to make sure that the old value is not populated again.
+                if post_failover_last_group_sync_time != last_group_sync_time:
+                    logger.info(
+                        "After failover - Verified: lastGroupSyncTime after failover is different from initial value."
+                    )
+                    break
+            logger.info(
+                "The value of lastGroupSyncTime in drpc is not updated after failover. Retrying."
+            )
+        logger.info(
+            f"The value of lastGroupSyncTime after failover is {post_failover_last_group_sync_time}."
+        )
+
+        # Verify lastGroupSyncTime after failover
+        time_format = "%Y-%m-%dT%H:%M:%SZ"
+        post_failover_last_group_sync_time_formatted = datetime.strptime(
+            post_failover_last_group_sync_time, time_format
+        )
+        current_time = datetime.strptime(
+            datetime.utcnow().strftime(time_format), time_format
+        )
+        time_since_last_sync = (
+            current_time - post_failover_last_group_sync_time_formatted
+        ).total_seconds() / 60
+        logger.info(
+            f"After failover - Time in minutes since the last sync is {time_since_last_sync}"
+        )
+        assert (
+            time_since_last_sync < 2.5 * scheduling_interval
```

Comment:
```suggestion
            time_since_last_sync < 2 * scheduling_interval
```

Reply:
Changed to 3 * scheduling_interval as discussed.